companies/onlinetvrecorder.json

```
@@ -0,0 +1,26 @@
+{
+    "slug": "onlinetvrecorder",
+    "relevant-countries": [
+        "de"
```

```suggestion
        "all"
```
I would suggest "all" as the service is available worldwide and in multiple languages.